CircuitPython_Templates/i2c_scan/code.py

```diff
@@ -1,15 +1,29 @@
 """CircuitPython I2C Device Address Scan"""
+# If you run this and it seems to hang, try manually unlocking
+# your I2C bus from the REPL with
+#  >>> import board
+#  >>> board.I2C().unlock()
+
```

Comment:
Please remove this, as it is explained in the guide page associated with this example.

Reply:
cool. thanks. i wanted to do that anyway. wasn't sure.